Formula/alan.rb

```diff
@@ -0,0 +1,17 @@
+class Alan < Formula
+  desc "Autoscalable Programming Language"
+  homepage "https://github.com/alantech/alan"
+  url "https://github.com/alantech/alan/releases/download/v0.1.34/alan-macos.tar.gz"
+  sha256 "1114cf7ceb5e9d2db92f2a3a38f9e05f33d9efdbb677b367a8e2405530b271e8"
+  license "Apache-2.0"
```

Comment:
can we omit the license? it is technically not Apache-2.0

Reply:
I don't think we can omit. Let's change it to AGPL-3.0

Reply:
Set it to AGPL-3.0